_rules/aria-attr-valid-5f99a7.md

```diff
@@ -5,11 +5,12 @@ rule_type: atomic
 description: |
   This rule checks that each aria- attribute specified is defined in ARIA 1.1
 accessibility_requirements:
-  wcag20:4.1.2: # Name, Role, Value (A)
+  aria11:host_general_attrs:
```

Comment:
I don't understand why we would link this rule to a section that has requirements for host languages. I think we need to find requirements for authors...

Reply:
ARIA doesn't neatly spell out its requirements like WCAG does. Nor does it have a clear distinction between what is a requirement for a host language, a content author or a "conformance checker" (which I believe is an ARIA test tool). The section this references is about using ARIA attributes in a language, which I think maps quite well to what this rule is about. If you have an alternative I'm happy to change it.

Reply:
(Comment from @kasperisager is on its way)

Reply:
So I did some digging and perhaps a simplified example might illustrate my findings and concerns. Imagine that ARIA defines the concept of tools and the process of using a screwdriver to put in a screw. What the rule in question then tries to do is to consider it an error for authors to use a hammer to put in a nail because ARIA doesn't specify "hammer" as a tool. However, ARIA also doesn't specify that using tools other than the ones defined by ARIA is an issue. Yes, it is quite a glaring issue if one attempts to use a flemgrestel for putting in a dulygoo, or use something that looks like a screwdriver, but isn't quite, for putting in a screw. 

The fact remains though that nowhere in ARIA is stated as a "**MUST**" author conformance requirement that only `aria-*` attributes defined by ARIA be used. In fact, Core AAM even specifies that user agents **SHOULD** allow authors to use `aria-*` attributes not defined by ARIA and map these to accessibility APIs regardless, for the purpose of forward compatibility (https://www.w3.org/TR/core-aam-1.1/#statePropertyMappingGeneralRules).

In summary, I don't see how we can list ARIA as an accessibility requirement for this rule. It's not that I don't want us to; I simply cannot justify it. For a rule such as "ARIA state or property allowed" it's more straight forward in light of https://github.com/act-rules/act-rules.github.io/pull/607#discussion_r298135912.

Also, this...

> Nor does it have a clear distinction between what is a requirement for a host language, a content author or a "conformance checker" (which I believe is an ARIA test tool).

...I don't quite agree with. ARIA has user agent, host language, and author conformance requirements littered throughout ("host languages **MUST**", "user agents **MUST**", and "authors **MUST**") and makes it clear that conformance checkers must issue errors when author requirements aren't met.

Reply:
I created an issue here. You've convinced me. I think we can solve this another way. I'll take this file change from the PR.